plugins/available/osx.plugin.bash

```diff
@@ -111,5 +111,78 @@ function refresh-launchpad() {
   fi
 }
 
+function list-jvms(){
+  about 'List java virtual machines and their states in macOS'
+  example 'list-jvms'
+  group 'osx'
+
+  JDKS_DIR="/Library/Java/JavaVirtualMachines"
+  JDKS=( $(ls ${JDKS_DIR}) )
+  JDKS_STATES=()
+
+  # Map state of JDK
+  for (( i = 0; i < ${#JDKS[@]}; i++ )); do
+    if [[ -f "${JDKS_DIR}/${JDKS[$i]}/Contents/Info.plist" ]]; then
+      JDKS_STATES[${i}]=enable
+    else
+      JDKS_STATES[${i}]=disable
+    fi
+    echo "${i} ${JDKS[$i]} ${JDKS_STATES[$i]}"
+  done
+}
+
+
+function pick-default-jvm(){
+  about 'Pick a default java virtual machines in system-wide scope in macOS'
+  example 'pick-default-jvm'
+
+  JDKS_DIR="/Library/Java/JavaVirtualMachines"
+  JDKS=( $(ls ${JDKS_DIR}) )
+  JDKS_STATES=()
+
+  # Map state of JDK
+  for (( i = 0; i < ${#JDKS[@]}; i++ )); do
+    if [[ -f "${JDKS_DIR}/${JDKS[$i]}/Contents/Info.plist" ]]; then
+      JDKS_STATES[${i}]=enable
+    else
+      JDKS_STATES[${i}]=disable
+    fi
+    echo "${i} ${JDKS[$i]} ${JDKS_STATES[$i]}"
+  done
+
+  # Declare variables
+  DEFAULT_JDK_DIR=""
+  DEFAULT_JDK=""
+  OPTION=""
+
+  # OPTION for default jdk and set variables
+  while [[ ! "$OPTION" =~ ^[0-9]+$ || OPTION -ge "${#JDKS[@]}" ]]; do
+    read -p "Enter Default JDK: "  OPTION
```

Comment:
You're mixing the terms "JVM" and "JDK" - let's use "JVM" consistently.

Please also change the wording so that it's clear that the user needs to type in the index of the JVM they want to use. With the current wording, it's not clear whether they need to type in the number or the full name.